### DISCUSSION TITLE
tune_ROSCO.py missing in latest commit (?)

### DISCUSSION THREAD

**Discussion by hdrengs** [NatLabRockies/ROSCO#243] [General]
Hi, 
Just cloned latest ROSCO version to my computer and see that the tune_ROSCO.py is not included in "Tune_cases" as before (e.g v 2.6.0). Cannot see that it is included anywhere else either. Is the ROSCO tuning performed from another file in the latest version or is the file simply missing? Any updates causing tune_cases.py from v.2.6.0 not to be compatible with v2.8.0?

**Comment by dzalkind** [comment]
Hi,

A similar functionality is included in this example: https://github.com/NREL/ROSCO/blob/main/Examples/03_tune_controller.py 

Best, Dan
